config/gnbsim.yaml

```diff
@@ -40,6 +40,9 @@ configuration:
               taiSliceSupportList:
                 - sst: 1 # Slice/Service Type (uinteger, range: 0~255)
                   sd: 010203 # Slice Differentiator (3 bytes hex string, range: 000000~FFFFFF)
+                - sst: 1
+                  sd: 000001
+                - sst: 2
```

Comment:
What happens if there is no sd ?

Reply:
It is not encoded in the NGSetupRequest message by Gnbsim.

Reply:
Could you please point me to code where its skipped ? Approving code, assuming we will not be dealing with situation immediately.

Reply:
[Optional field Sd decoding at NGAP ](https://github.com/omec-project/ngap/blob/main/ngapConvert/SNssai.go#L27)